batcher/aligned-batcher/src/lib.rs

```diff
@@ -1002,15 +995,17 @@ impl Batcher {
         let fee_per_proof = U256::from(gas_per_proof) * gas_price;
         let fee_for_aggregator = (U256::from(AGGREGATOR_GAS_COST)
             * gas_price
-            * U256::from(DEFAULT_AGGREGATOR_FEE_MULTIPLIER))
-            / U256::from(DEFAULT_AGGREGATOR_FEE_DIVIDER);
+            * U256::from(DEFAULT_AGGREGATOR_FEE_PERCENTAGE_MULTIPLIER))
+            / U256::from(PERCENTAGE_DIVIDER);
         let respond_to_task_fee_limit = (fee_for_aggregator
-            * U256::from(RESPOND_TO_TASK_FEE_LIMIT_MULTIPLIER))
-            / U256::from(RESPOND_TO_TASK_FEE_LIMIT_DIVIDER);
+            * U256::from(RESPOND_TO_TASK_FEE_LIMIT_PERCENTAGE_MULTIPLIER))
+            / U256::from(PERCENTAGE_DIVIDER);
+        let final_gas_price = gas_price * U256::from(GAS_PRICE_PERCENTAGE_MULTIPLIER)
```

Comment:
`final_gas_price` does not tell anything about the modification applied to the gas price

Also we should discuss where should be done this kind of actions because, the incremented gas price is being propagated across the function calls, but `fee_params.gas_price` does not say anything about a previous increment of 10%

Reply:
IMO, gas price increment of 10% should be done here

```rust 
//eth/mod.rs

pub async fn try_create_new_task(
...
) ... {  
...
.gas_price(fee_params.gas_price); <- here
...
}
```

Reply:
One more thing, with the new system of retries, this can change too